test/testHelpers/collections/collectionHelper.ts

```diff
@@ -26,8 +26,16 @@ export const createCollectionPayload = (): CollectionPayload => {
     alias: COLLECTION_ALIAS_STR,
     name: COLLECTION_NAME_STR,
     affiliation: COLLECTION_AFFILIATION_STR,
-    description: COLLECTION_DESCRIPTION_STR,
+    description: COLLECTION_DESCRIPTION_HTML,
     isPartOf: { type: DvObjectType.DATAVERSE, identifier: 'root', displayName: 'Root' }
   }
   return collectionPayload
 }
+export const transformedCollectionModel: CollectionPayload = {
```

Comment:
think we could use the `createCollectionModel` instead of creating a new `transformedCollectionModel`, since all collection models should be transformed